### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by nick-george** [elastic/puppet-logstash#403]
Looking at the code frequency on this puppet module, it appears that the project is dead. Any chance Elastic could hand over control of the project to someone who might be willing/able to maintain it? (Vox Pupuli comes to mind)

**Comment by jarpy** [comment]
Maybe a question for you, @nkammah?

**Comment by nkammah** [comment]
@nick-george you are right that we have not been able to invest time to prioritize work on the puppet-logstash and I don't anticipate this to change in the near future. I was not aware of Vox Pupuli but this sounds like a very reasonable approach. Do you happen to know what such a transition would look like ?

**Comment by nick-george** [comment]
I'm afraid I don't, but it might be worth reaching out to Vox Pupuli (apparently a team of people, not an individual) to see if they'd have any interest in sponsoring the project.

**Comment by Fabian1976** [comment]
Or at least publish a new version on puppetforge with a stdlib > 6 dependency. Everyone is forking this repo now and only update that dependency.

**Comment by shaneshort** [comment]
I'd like to echo the above sentiment, is there anything we can do to assist getting this module shifted over to VP?

**Comment by alexjfisher** [comment]
@nkammah Vox Pupuli would be happy to help look after this project.

Have you seen https://voxpupuli.org/docs/migrate_module ?

If you would like to proceed with just this module or potentially some of your other puppet modules too, let me know and I'll add you to the Vox Pupuli org so that you can perform the migration.

**Comment by nkammah** [comment]
@alexjfisher i apologize for the radio silence - we've been discussing the situation internally and I will be able to provide an update soon, I'd expect within 7 days. Thanks!

**Comment by Rocco83** [comment]
@nkammah it's actually my fault, given https://github.com/elastic/puppet-logstash/issues/403#issuecomment-768216488 i was reaching in chat the admin team.
Hope that this module can get life back soon.

**Comment by Rocco83** [comment]
@nkammah news?

**Comment by nkammah** [comment]
We’ve discussed it internally and have decided to officially stop development and will be releasing the project into the community - we're looking into our next steps. My apologies for not providing an update within the 7 days I had mentioned originally.

**Comment by nkammah** [comment]
We're starting the process to hand-off the 4 puppet modules to Vox Pupuli.

**Comment by nkammah** [comment]
The 4 Elastic puppet modules have been handed-off to Vox Pupuli - many thanks for adopting the modules.

**Comment by pingram3030** [comment]
Great, this repo was thrown over the fence and it looks like there's no one at the helm.

How do we go about contributing, in regards to merging code and fixing up breakages, to try and get this repo up to snuff?

**Comment by smortex** [comment]
@pingram3030 the module has been donated to voxpupuli.

First sted would be to use modulesync to setup the repository in a way GitHub does CI, and ensure the test pass.  Then submitting PR should be smooth… Anyone can step in (I am not a user of logstash so only did it for [ElasticSearch](https://github.com/voxpupuli/puppet-elasticsearch/pull/1125) and [Kibana](https://github.com/voxpupuli/puppet-kibana/pull/52)).

See https://github.com/voxpupuli/modulesync_config/ for updating the module with modulesync.  Feel free to reach #voxpupuli on IRC for chatting / help and advises :wink:

**Comment by bastelfreak** [comment]
I did a bit of cleanup and an initial modulesync in https://github.com/voxpupuli/puppet-logstash/pull/408. it doesn't look too bad, someone just needs to cleanup the acceptance tests and probably manage the whole spec_helper_acceptance via modulesync as well.
